eng/illink.targets

```diff
@@ -281,7 +281,8 @@
     <!-- When running from Desktop MSBuild, DOTNET_HOST_PATH is not set.
          In this case, explicitly specify the path to the dotnet host. -->
     <PropertyGroup Condition=" '$(DOTNET_HOST_PATH)' == '' ">
-      <_DotNetHostDirectory>$(DotNetRoot)</_DotNetHostDirectory>
+      <!-- This is defined when building in Visual Studio. -->
+      <_DotNetHostDirectory>$(NetCoreRoot)</_DotNetHostDirectory>
```

Comment:
We should also update it here - https://github.com/dotnet/runtime/blob/83b4250d0f07c87c25d32365f7e697cc6e66cf43/src/libraries/illink-sharedframework.targets#L76.

Reply:
Thanks, will update.